sw/host/provisioning/cp_lib/src/lib.rs

```diff
@@ -191,14 +172,18 @@ pub fn unlock_raw(opts: &Opts, transport: &TransportWrapper) -> Result<()> {
 }
 
 pub fn run_sram_cp_provision(
-    opts: &Opts,
     transport: &TransportWrapper,
+    jtag_params: &JtagParams,
+    reset_delay: &Duration,
+    sram_program: &SramProgramParams,
+    provisioning_actions: &ManufCpProvisioningActions,
     provisioning_data: &ManufCpProvisioningData,
+    timeout: &Duration,
```

Comment:
nit: `Duration` is a relatively small struct and is `Copy`.  You don't need to take this argument by reference here and deref it in the block of code below.

Reply:
fixed.